docs/providers/cassandra.md

```diff
@@ -34,3 +34,11 @@ featureform apply cassandra_config.py --host $FEATUREFORM_HOST
 ```
 
 We can re-verify that the provider is created by checking the [Providers tab of the Feature Registry](../getting-started/exploring-the-feature-registry.md).
+
+### Updatable Configuration Fields
```

Comment:
I wonder if we'd want to call them "Mutable" -- I know "Updatable" is more accurate but it also feels clunky.